tensorboard/manager_test.py

```diff
@@ -268,6 +268,24 @@ def setUp(self):
   def _list_info_dir(self):
     return os.listdir(self.info_dir)
 
+  def assertMode(self, path, expected):
+    """Assert that the permission bits of a file are as expected.
+
+    Args:
+      path: File to stat.
+      expected: `int`; a subset of 0o777.
+
+    Raises:
+      AssertionError: If the permissions bits of `path` do not match
+        `expected`.
+    """
+    stat_result = os.stat(path)
+    format_mode = lambda m: "0o%03o" % m
```

Comment:
Could just use `oct()`?

Reply:
I had that, but I prefer the leading zeros (`"0o0"` is not clearly a
file mode, whereas `"0o000"` is). Also, `oct` is different on Python 2
vs. Python 3.